ocs/utils.py

```diff
@@ -303,8 +302,8 @@ def setup_deb_cdn_repo(node, build=None):
     user = 'redhat'
     passwd = 'OgYZNpkj6jZAIF20XFZW0gnnwYBjYcmt7PeY76bLHec9'
     num = build.split('.')[0]
-    cmd = 'umask 0077; echo deb https://{user}:{passwd}@rhcs.download.redhat.com/{num}-updates/Tools ' \
-          '$(lsb_release -sc) main | tee /etc/apt/sources.list.d/Tools.list'.format(user=user, passwd=passwd, num=num)
+    cmd = ('umask 0077; echo deb https://{user}:{passwd}@rhcs.download.redhat.com/{num}-updates/Tools '
```

Comment:
Please go over the rest of patch and try to ensure shorter lines you are changing.

We should do refactoring later on and force some line length for pep8 but now please do it at least for lines you are changing. Thanks

Reply:
> Please go over the rest of patch and try to ensure shorter lines you are changing.

As you correctly state, we don't enforce the line length. This PR is focused on one aspect of the code policy only. You can fix it for the whole project if you really care in a separate pull request.

Reply:
I meant that we don't enforce it in flake8 yet cause of we took the code from ceph-ci which didn't had this convention but in our [guidelines](https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md) we have this statement:
`Line length should be maximum 79 characters!`

So I believe that for new coming code we should follow this. And as you are already touching this code you should also follow the parentheses  convention as I stated in some comments.

Or why do we have those conventions defined?

It's on the list of the things to do the refactoring of all the old code from ceph-ci and change line length but it's with low priority now.

I don't care and can approve, just what I said, we should follow the conventions defined.

Reply:
Ah, ok. I will fix that in this pull request then.

I agree that we should follow a guidelines and common practices, esp. because we have big gaps there. So read this pull request as a attempt to partially address that. And given the current state of the project, I don't believe we can maintain the consistency without static checks.

So eg. in case of line length, my impression was that this is not actually enforced yet (no matter what is stated in the code guidelines), neither in manual code review, nor in tox. See eg. the list such issues added into the repo in this month:

```
$ tox -e flake8 | awk --field-separator=: '/^\./{ system("git blame " $1 " -L " $2 "| head -1"); }' | grep 2019-06
a7a6e18c (Coady LaCroix    2019-06-05 10:02:34 -0700 20)         'reportportal-client @ git+https://github.com/reportportal/client-Python.git@master',
c9fc4fd3 (Akarsha Rai 2019-06-15 00:55:21 +0530 270)         command=f"dd if=/dev/urandom of=/var/lib/www/html/dd_a bs={bs} count={count}"
916bb57f (Zack Cerza    2019-06-07 13:20:09 -0600  745)         log.debug(f"Installer exists ({installer_binary_path}), skipping download.")
700bd748 (Andrew Schoen 2019-06-08 09:36:22 -0500  18) from utility.utils import destroy_cluster, run_cmd, get_openshift_installer, get_openshift_client, is_cluster_running
0d27fe4d (Andrew Schoen 2019-06-18 07:53:38 -0500  55)         log.info("Attempting teardown of non-accessible cluster: %s", cluster_path)
0d27fe4d (Andrew Schoen 2019-06-18 07:53:38 -0500  58)         msg = "The given cluster can not be connected to: {}. ".format(cluster_path)
0d27fe4d (Andrew Schoen 2019-06-18 07:53:38 -0500  59)         msg += "Provide a valid --cluster-path or use --deploy to deploy a new cluster"
8c8e6fd2 (Andrew Schoen 2019-06-13 12:19:53 -0500  62)         log.info("A testing cluster will be deployed and cluster information stored at: %s", cluster_path)
9dcda706 (Andrew Schoen 2019-06-14 07:14:29 -0500 126)     create_oc_resource('common.yaml', cluster_path, _templating, config.ENV_DATA)
9dcda706 (Andrew Schoen 2019-06-14 07:14:29 -0500 170)         'operator-openshift-with-csi.yaml', cluster_path, _templating, config.ENV_DATA
9dcda706 (Andrew Schoen 2019-06-14 07:14:29 -0500 186)     create_oc_resource('cluster.yaml', cluster_path, _templating, config.ENV_DATA)
0b5601c8 (Andrew Schoen 2019-06-20 11:50:19 -0500 188)     POD = ocp.OCP(kind=constants.POD, namespace=config.ENV_DATA['cluster_namespace'])
0b5601c8 (Andrew Schoen 2019-06-20 11:50:19 -0500 190)         kind=constants.CEPHFILESYSTEM, namespace=config.ENV_DATA['cluster_namespace']
9dcda706 (Andrew Schoen 2019-06-14 07:14:29 -0500 213)     create_oc_resource('toolbox.yaml', cluster_path, _templating, config.ENV_DATA)
e4c80de2 (Elad Ben Aharon  2019-06-06 12:19:46 +0300  33)     return f"{resource_type}-{resource_description[:23]}-{current_date_time[:10]}"
63ee6c7e (Sidhant Agrawal  2019-06-14 00:03:33 +0530 139)     cbp_data['metadata']['name'] = pool_name if pool_name else create_unique_resource_name(
0477cbf7 (prsurve          2019-06-14 19:12:41 +0530 170)         sc_data['parameters']['csi.storage.k8s.io/node-publish-secret-name'] = secret_name
0477cbf7 (prsurve          2019-06-14 19:12:41 +0530 171)         sc_data['parameters']['csi.storage.k8s.io/node-publish-secret-namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
0477cbf7 (prsurve          2019-06-14 19:12:41 +0530 174)         sc_data['parameters']['csi.storage.k8s.io/node-stage-secret-name'] = secret_name
0477cbf7 (prsurve          2019-06-14 19:12:41 +0530 175)         sc_data['parameters']['csi.storage.k8s.io/node-stage-secret-namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
63ee6c7e (Sidhant Agrawal  2019-06-14 00:03:33 +0530 186)     sc_data['metadata']['name'] = sc_name if sc_name else create_unique_resource_name(
96d08981 (Elad Ben Aharon  2019-06-11 02:49:07 +0300 190)     sc_data['parameters']['csi.storage.k8s.io/provisioner-secret-name'] = secret_name
96d08981 (Elad Ben Aharon  2019-06-11 02:49:07 +0300 191)     sc_data['parameters']['csi.storage.k8s.io/provisioner-secret-namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
63ee6c7e (Sidhant Agrawal  2019-06-14 00:03:33 +0530 218)     pvc_data['metadata']['name'] = pvc_name if pvc_name else create_unique_resource_name(
916bb57f (Zack Cerza       2019-06-07 13:20:09 -0600 19) POD = ocp.OCP(kind=constants.POD, namespace=config.ENV_DATA['cluster_namespace'])
916bb57f (Zack Cerza      2019-06-07 13:20:09 -0600 44)     self.fs_data['metadata']['namespace'] = config.ENV_DATA['cluster_namespace']
916bb57f (Zack Cerza  2019-06-07 13:20:09 -0600 25)         "You can easily access data from config.ENV_DATA like cluster_name: %s",
96d08981 (Elad Ben Aharon 2019-06-11 02:49:07 +0300 98)     pod_data['spec']['volumes'][0]['persistentVolumeClaim']['claimName'] = class_instance.pvc_obj.name
916bb57f tests/manage/test_ocs_336_346.py (Zack Cerza       2019-06-07 13:20:09 -0600  41)     self.fs_data['metadata']['namespace'] = config.ENV_DATA['cluster_namespace']
c9fc4fd3 (Akarsha Rai 2019-06-15 00:55:21 +0530 141)         pod_data['spec']['volumes'][0]['persistentVolumeClaim']['claimName'] = pvc_obj.name
96d08981 (Elad Ben Aharon 2019-06-11 02:49:07 +0300 42)             f"PVC {self.pvc_obj.name} failed to reach status {defaults.STATUS_TERMINATING}"
87a296f9 unittests/test_defaults.py (vasukulkarni    2019-06-10 10:25:36 -0700 6)     assert defaults.CEPHFILESYSTEM_DICT['spec']['metadataPool']['replicated']['size'] == 3
0fb59002 (Andrew Schoen 2019-06-07 16:19:33 -0500  52)         help="If provided the test cluster will be destroyed after tests complete",
8c8e6fd2 (Andrew Schoen 2019-06-13 12:19:53 -0500  59)         help="If provided a test cluster will be deployed on AWS to use for testing",
7aab6e81 (Andrew Schoen 2019-06-19 16:24:56 -0500 120)     ocsci_config.RUN['cli_params']['teardown'] = get_cli_param(config, "teardown", default=False)
7aab6e81 (Andrew Schoen 2019-06-19 16:24:56 -0500 121)     ocsci_config.RUN['cli_params']['deploy'] = get_cli_param(config, "deploy", default=False)
```

Maybe we should have added some warning (without failing the flake8 check) or limit the check to certain directories.